api/src/env.ts

```diff
@@ -291,6 +291,14 @@ const defaults: Record<string, any> = {
 
 	FLOWS_EXEC_ALLOWED_MODULES: false,
 	FLOWS_ENV_ALLOW_LIST: false,
+
+	PRESSURE_LIMITER_ENABLED: true,
```

Comment:
I personally think this should be an opt-in feature instead of opt-out as misconfiguring this can impact the availability and user experience of Directus instances.